src/background/providers/cloudflare.ts

```diff
@@ -288,19 +289,23 @@ export class CloudflareProvider implements Provider {
         }
 
         const hasBodyParams = QUALIFIED_BODY_PARAMS.every((param) => {
-            return details.requestBody !== null && param in details.requestBody.formData!;
+            return (
+                details.requestBody &&
+                details.requestBody.formData &&
+                param in details.requestBody.formData
+            );
         });
         if (!hasBodyParams) {
             return;
         }
 
         const flattenFormData: { [key: string]: string[] | string } = {};
         for (const key in details.requestBody.formData) {
-            if (details.requestBody.formData[key].length == 1) {
-                const [value] = details.requestBody.formData[key];
-                flattenFormData[key] = value;
+            if (details.requestBody.formData[key as string].length == 1) {
```

Comment:
This seems to be a vulnerability rather than a lint error. https://github.com/nodesecurity/eslint-plugin-security/blob/master/docs/the-dangers-of-square-bracket-notation.md

This patch make the linter pass rather fixing the vulnerability. https://stackoverflow.com/questions/51273469/how-to-resolve-eslint-generic-object-injection-sink-error

Reply:
I think defining a function checking for the key of `constructor`, `__proto__`, and `prototype` should be fine.